### DISCUSSION TITLE
Unable to run kratos migrate command and server command

### DISCUSSION THREAD

**Discussion by ganeshkbhat** [ory/kratos#1476] [Q&A] [answered]
I am using the scoop installed kratos executable using link `https://www.ory.sh/kratos/docs/next/install`. I am presuming it is a sqlite build. I am getting the following error when running the migrate command 

`kratos migrate sql -e DSN=sqlite://db.sqlite --yes`

```
PS E:\workspace\projects\openapi-gen\golangmiddlewares\kratos> kratos migrate sql -e DSN=sqlite://db.sqlite --yes
time=2021-06-30T17:08:53+05:30 level=fatal msg=dsn must be set audience=application service_name=Ory Kratos service_version=v0.6.3-alpha.1
```

Second, What is the command to run the kratos serve correctly. I am looking at the following link. `https://www.ory.sh/kratos/docs/reference/configuration/`

Is there test sample configuration for the above `.yml` file.

**Comment by aeneasr** [comment, marked as answer]
The environment variables are not used correctly. The correct syntax is (when using cmd.exe):

```
set DSN=sqlite://db.sqlite
kratos migrate sql -e --yes
```

For more info, please check online resources, for example: https://www3.ntu.edu.sg/home/ehchua/programming/howto/Environment_Variables.html

**Reply by ganeshkbhat** [reply]
:-) Bad me.  Whch binary do I use for sqlite? Currently using the scoop installed executable. Seems like I am getting a sqlite error

`time=2021-06-30T18:08:21+05:30 level=warning msg=Unable to connect to database, retrying. audience=application error=map[message:unsupported dialect 'sqlite3'] service_name=Ory Kratos service_version=v0.6.3-alpha.1`

Can you also help me with the `kratos serve` command and the .yml file sample. I see this and see that there are lot .yml file concat in https://github.com/ory/kratos/blob/master/test/e2e/run.sh

**Reply by vinckr** [reply]
@ganeshkbhat  just checking in if your problem still persists. 
What was your question about [kratos serve](https://www.ory.sh/kratos/docs/next/cli/kratos-serve/)?
For an example configuration, I recommend the [quickstart kratos.yml](https://github.com/ory/kratos/tree/master/contrib/quickstart/kratos/email-password).
I will mark this as answered for now, but feel free to come back to the discussion any time.
